cdc/cdc/sink/tikv.go

```diff
@@ -281,14 +290,18 @@ func extractEntry(entry *model.RawKVEntry, now uint64) (opType model.OpType, key
 	return
 }
 
-func (b *tikvBatcher) Append(entry *model.RawKVEntry) {
+func (b *tikvBatcher) Append(entry *model.RawKVEntry) error {
 	log.Debug("[TRACE] tikvBatch::Append", zap.Any("event", entry))
 
 	if len(b.Batches) == 0 {
 		b.now = b.getNow()
 	}
 
-	opType, key, value, ttl := extractEntry(entry, b.now)
+	opType, key, value, ttl, err := extractEntry(entry, b.now)
+	if err != nil {
+		log.Error("failed to extract entry", zap.Any("event", entry), zap.Error(err))
+		return err
```

Comment:
Suggest to do the statistics (`k.statistics.AddInvalidKeyCount()`) here.

Here we know it's a invalid key error. But it would be confused if we `AddInvalideKeyCount` when `batcher.Append` return error.